google-cloud-core/Gemfile

```diff
@@ -3,10 +3,8 @@ source "https://rubygems.org"
 gemspec
 
 gem "rake", "~> 11.0"
-gem "google-api-client", "~> 0.9.11"
-gem "grpc", "~> 1.0"
-gem "google-protobuf", "~> 3.0"
-gem "googleapis-common-protos", "~> 1.3"
+gem "google-api-client", "~> 0.10.0"
+gem "google-gax", "~> 0.8.0"
```

Comment:
Its replacing the `grpc` and `google-protobuf` and `googleapis-common-protos` dependencies, the same way that those direct dependencies have been removed from gemspec files. The `google-gax` gem declares all those dependencies, and we should use the versions that it specifies.